templates/zerver/integrations.html

```diff
@@ -1009,6 +1009,48 @@
 
     </div>
 
+    <div id="hellosign" class="integration-instructions">
+
+      <p>
+        Configuring the HelloSign integration is easy, first, create a stream
+        (we recommend naming it <code>hellosign</code>), if no stream is supplied,
+        the default is hellosign if it exists. This does not mean you don't
+        have to establish a stream first.
+      </p>
+
+      <p>
+        Next, go to the Zulip settings page and create a bot named HelloSign.
+        Note the bot API key and add it to the URL below. If you want your
```

Comment:
I think adding the API key should be next to the URL in question. Or giving the URL a name, like `the integration URL` and then clearly naming it in the doc.